CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 Change Log
 ==========
 
+Version 1.1.3 *(2017-10-02)*
```

Comment:
Sorry about that, but let's not edit Changelog in PRs, this will confuse users because the version is not released yet and date will be fixed only when release is done :)

We'll edit that before merging, no changes required from you, JFYI :)